packages/design-system/src/styles/components/_SingleInputDateField.scss

```diff
@@ -67,12 +67,7 @@
   border: 2px solid transparent;
 }
 
-.rdp-button[aria-disabled='true'] {
-  opacity: 0.25;
-  pointer-events: none;
-}
-
-.rdp-button:not([aria-disabled='true']) {
+.rdp-button:not(:disabled) {
   cursor: pointer;
```

Comment:
Could be redundant; I'll check

Reply:
Yeah, they have a button reset rule of their own. That could be something to revisit later—whether we want to try to optimize their CSS. At the beginning, our strategy was to try to keep their original CSS mostly intact and add overrides to it so it would be easy to upgrade the library as time went on. With that strategy, though, we do include a lot more rules than we need.